### PR TITLE
Refactoring files into modules by action

### PR DESCRIPTION
**Pull request by diestrin**

### CODE REVIEW

**Comment by diestrin** [comment]
Use `Squash and merge` if everything looks good please

**Review by ecairol — APPROVED** [review]

File: src/state.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  bid(state, action) {
+    const tradeRate = action.payload;
+    const operation = state.balance.base / tradeRate;
+    const fee = operation * feeRate;
+    const result = operation - fee;
+    const feeInBase = fee * tradeRate;
+
+    transaction({
+      isBid: true,
+      tradeRate, result, feeInBase,
```
Comment (jrgherrera) [inline src/state.js]: transaction method does not receive a property called feeInBase, it receives a property called fee

**Review by jrgherrera — CHANGES_REQUESTED** [review]
Please fix the property in the transaction call

**Comment by diestrin** [comment]
@jrgherrera sir yes sir, work is done :)

**Review by jrgherrera — APPROVED** [review]

**Pull request merged**